src/event/share-event.js

```diff
@@ -1,15 +1,21 @@
 /**
  * Share plugin event types.
  */
-const ShareEvent = {
+export const ShareEvent = {
   /**
-   * Fired when the user clicks on share icon.
+   * Fired when the user clicks on share plugin icon.
    */
-  SHARE_CLICKED: 'share_clicked',
+  SHARE_EMBED_OPEN: 'Share_embed_open',
```

Comment:
whats the advantage of explicitly adding SHARE_EMBED to the name ?

Reply:
The names are from the ticket

Reply:
oh, it's to match the value. i see